_education/advanced-training-and-certifications/entrepreneurship-training.md

```diff
@@ -20,8 +20,23 @@ template: 6-info-page
 
 If you qualify for any of the GI Bill or educational assistance programs, and you are interested in starting or improving a business, VA provides entrepreneurship training through its Center for Veterans Enterprise and the Small Business Administration.
 
-Learn more about [entrepreneurship training](http://www.benefits.va.gov/GIBILL/docs/factsheets/Entrepreneurship_Training.pdf).
+You do not have to use GI Bill funding to participate in this program. 
 
+### Are you eligible for this training?
+
+Yes, if you are eligible for any of the following:
+
+- Montgomery GI Bill, [Active Duty](/education/gi-bill/montgomery-active-duty) or [Selected Reserve](/education/gi-bill/montgomery-selected-reserve)
+- [Post-9/11 GI Bill](/education/gi-bill/post-9-11)
+- [VEAP](/education/legacy-programs/veap)
```

Comment:
@emilyville No technical fix needed here, it's simply a wrong link. VEAP is no longer at `/education/legacy-programs/veap` but is at `/education/other-educational-assistance-programs/veap/`.  So changing this line should make the PR green.